### ISSUE TITLE
(QUESTION) Can you install it on Wii U?

### ISSUE THREAD

**Issue by LukahF** [rverseTeam/rverse2#51]
Can you install rvers on Wii U? If yes, how?

**Comment by kamilthecitratester** [comment]
yes but use "pretendo martini" to apply the patch

**Comment by sladewatkins** [comment]
No, as rverse2 is 3DS-only. rverse3 is currently in testing and will support both 3DS *and* Wii U. We look forward to launching rverse3 sometime down the road. :)
